singularity-eos/eos/eos_stellar_collapse.hpp

```diff
@@ -220,6 +220,20 @@ class StellarCollapse : public EosBase<StellarCollapse> {
                                        bool dependent_var_log);
 
  private:
+  class LogT {
+   public:
+    PORTABLE_INLINE_FUNCTION
+    LogT(const DataBox &field, const Real Ye, const Real lRho)
+        : field_(field), Ye_(Ye), lRho_(lRho) {}
+    PORTABLE_INLINE_FUNCTION Real operator()(const Real lT) const {
+      return field_.interpToReal(Ye_, lT, lRho_);
+    }
+
+   private:
+    const DataBox &field_;
```

Comment:
I see that this is mostly a copy/paste move, but be warned of potentially cryptic compilation errors due to const class members.

Reply:
In this case it should be ok, and it was a copy/paste, but point taken. Will remove the `const`s.

Reply:
This actually has to be `const` due to the `const` correctness elsewhere in the eos object.